sodium/src/pwhash.rs

```diff
@@ -54,23 +54,25 @@ pub fn hash(
             ops_limit as libc::c_ulonglong,
             mem_limit,
             alg as libc::c_int,
-        );
+        )
```

Comment:
from the docs
> The function returns 0 on success, and -1 if the computation didn't complete, usually because the operating system refused to allocate the amount of requested memory.

and the [sys bindings](https://docs.rs/rust_sodium-sys/0.10.2/rust_sodium_sys/fn.crypto_pwhash.html) look like they return that integer... maybe we should be grabbing it to make sure

Reply:
Ok. I'll add an unreachable!